docs/_templates/meta.html

```diff
@@ -13,7 +13,7 @@
     </a>
 
 </div>
-<div class="facebook-terms">
+<div class="meta-terms">
     <ul>
         <li>
             <a href="https://opensource.facebook.com/legal/terms">Terms of Use</a>
```

Comment:
TOU and Privacy are still fb?

Reply:
Yep, the pages have the Meta header logo, but this is what OSS lint tool is expecting/requiring.